doc/sailfish-webview-webviewexperimental.qdoc

```diff
@@ -0,0 +1,181 @@
+
+See also: \l{WebView::contentWidth}, \l{WebView::contentHeight},
+\l{WebView::viewportWidth}, \l{WebView::viewportHeight},
+\l{WebView::scrollableSize}
+
+\section2 WebView::contentWidth
+
+\c{qreal}-type read-only property.
+
+The width of the bounding box rectangle defined by the loaded content
+being displayed by the webview.
+
+See also: \l{WebView::contentRect}
+
+\section2 WebView::contentHeight
+
+\c{qreal}-type read-only property.
+
+The height of the bounding box rectangle defined by the loaded content
+being displayed by the webview.
+
+See also: \l{WebView::contentRect}
+
+\section2 WebView::scrollableSize
+
+\c{QSizeF}-type read-only property.
+
+The geometric size of the content being displayed by the webview,
+in device pixels.
+
+The loaded page content is likely to fill an area much larger than the
+viewport of the webview (thus requiring the user to scroll the viewport in
+order to see content further down the page, etc).
+
+This property contains the total size of the content, in device pixels.
+
+See also: \l{WebView::contentRect}, \l{WebView::scrollableOffset}
+
+\section2 WebView::scrollableOffset
+
+\c{QPointF}-type read-only property.
+
+The (x and y pixel) offset into the scrollable size which describes
+the top-left pixel of the content being displayed by the viewport.
+
+The value of this property is effectively equal to the value of the
+x and y values of the \l{WebView::contentRect} after applying the appropriate
+\l{resolution} scale factor (to map from CSS pixel coordinates to
+device-resolution pixel coordinates).
+
+See also: \l{WebView::contentRect}, \l{WebView::scrollableSize},
+\l{WebView::resolution}
+
+\section2 WebView::resolution
+
+\c{float}-type read-only property
+
+The scaling factor which is applied to the (CSS-pixel) contentRect
+values to map to the appropriate device-resolution pixel coordinates.
+
+The value of this property is equal to the width of the device's screen
+divided by the width of the \l{WebView::contentRect}.  As such, it will update
+when the user zooms in or out (thus changing the width, in CSS pixels, of
+the contentRect), or if the page orientation changes (resulting in a change
+to landscape orientation from portrait orientation, or vice-versa).
+
+See also: \l{WebView::contentRect}
+
+\section2 WebView::horizontalScrollDecorator
+
+\l{QmlMozScrollDecorator}-type read-only property.
+
+The horizontal scroll bar.
+
+The QmlMozScrollDecorator type is available from the
+Qt5Mozilla import namespace.
+
+\section2 WebView::verticalScrollDecorator
+
+\l{QmlMozScrollDecorator}-type read-only property.
+
+The vertical scroll bar.
+
+The QmlMozScrollDecorator type is available from the
+Qt5Mozilla import namespace.
+
+\section2 WebView::painted
+
+\c{bool}-type read-only property.
+
+Whether the first paint has completed.
+
+\section2 WebView::parentId
+
+\c{int}-type read-only property.
+
+The uniqueId of the parent window (web view instance) which requested
+opening this location.
```

Comment:
See also `openUrlInNewWindow` or is that unrelated?

Reply:
it's related, but marked as internal currently.

Reply:
Ah, good point. Thanks.